src/js/utils/in-interaction-event.ts

```diff
@@ -0,0 +1,6 @@
+import { GenericObject } from 'types/generic.type';
+
+export function inInteraction(event: GenericObject | undefined): boolean {
```

Comment:
`event` is optional. When it is passed in, the expected type is `Event`.

```suggestion
export function inInteraction(event?: Event): boolean {
```

Reply:
That is what this means in the above context (as an argument as apposed to variable assignment). Event or undefined.

Reply:
Agreed, making this change now